src/error.rs

```diff
@@ -267,6 +269,8 @@ pub(crate) enum ErrorCode {
     Message,
     #[cfg(feature = "std")]
     Io(io::Error),
+    #[cfg(all(not(feature = "std"), feature = "unsealed_read_write"))]
```

Comment:
I'd suggest to use `#[allow(unused)]` here instead.
(The reasoning is that writing these feature bounds correctly will quickly become too complex)

Reply:
sounds fair